tests/publiccloud/create_aistack_env.pm

```diff
@@ -28,38 +28,43 @@ use File::Basename;
 use version_utils;
 use Data::Dumper;
 
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
+}
+
 sub install_dependency_package {
     my ($instance) = @_;
     my $rke2_url = get_var('RKE2_URL');
     my $kubectl_url = get_var('KUBECTL_URL');
     my $helm_url = get_var('HELM_URL');
 
     record_info('Dep pkg install');
-    trup_call("pkg install curl git docker");
+    trup_call("pkg install curl git podman python311");
 
     # docker activation section
     process_reboot(trigger => 1);
-    systemctl("enable docker");
-    systemctl("start docker");
-    systemctl("status docker");
+    systemctl("enable podman");
+    systemctl("start podman");
+    systemctl("status podman");
 
     # rke2 activation section
-    script_run("curl -sSL $rke2_url -o ./install_rke2.sh && chmod 775 ./install_rke2.sh");
-    script_run("sh ./install_rke2.sh");
-    script_run("echo 'export PATH=\$PATH:/opt/rke2/bin' >> ~/.bashrc");
+    assert_script_run("curl -sSL $rke2_url -o ./install_rke2.sh && chmod 775 ./install_rke2.sh");
+    assert_script_run("sh ./install_rke2.sh");
+    assert_script_run("echo 'export PATH=\$PATH:/opt/rke2/bin' >> ~/.bashrc");
+    assert_script_run("source ~/.bashrc");
     systemctl("enable rke2-server.service");
     systemctl("start rke2-server.service");
     systemctl("status rke2-server.service");
-    script_run("rke2 --version");
+    assert_script_run("rke2 --version");
 
     # helm activation section
-    script_run("curl -sSL $helm_url -o ./install_helm.sh && chmod 775 ./install_helm.sh");
-    script_run("sh ./install_helm.sh");
+    assert_script_run("curl -sSL $helm_url -o ./install_helm.sh && chmod 775 ./install_helm.sh");
+    assert_script_run("sh ./install_helm.sh");
     script_run("helm version");
 
     # kubectl activation section
-    script_run("curl -sSL $kubectl_url -o ./kubectl && chmod +x ./kubectl");
-    script_run("sudo mv ./kubectl /usr/local/bin/");
+    assert_script_run("curl -sSL $kubectl_url -o ./kubectl && chmod +x ./kubectl");
+    assert_script_run("sudo mv ./kubectl /usr/local/bin/");
```

Comment:
You're already root, so no need for `sudo` and you may do this instead:

```suggestion
    assert_script_run("curl -sSL $kubectl_url -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl");
```

Reply:
Fixed